src/classes/STG_SettingsManager_CTRL.cls

```diff
@@ -102,7 +107,7 @@ public with sharing class STG_SettingsManager_CTRL extends STG_Panel {
     * @description Returns whether the running user is a System Administrator.
     * @return void
     */
-    private boolean runningUserIsAdmin() {
+    public static boolean runningUserIsAdmin() {
```

Comment:
Since this method is now being changed to `public`, there is a similar method in `USER_UserService.isSysAdmin`. The logic implemented by these two methods are quite different however. @njjc Wondering if the method name on one or both of these should be changed to further clarify how the methods differ; and possibly if this method should be moved to `USER_UserService`. What do you think?

Reply:
Better names and centralizing in the service sounds like a good idea to me!

Reply:
done and pushed.